src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/ContainerProcessor.java

```diff
@@ -71,16 +80,30 @@ private void processBlob(BlobClient blob) {
         envelopeService
             .findEnvelope(blob.getBlobName(), blob.getContainerName())
             .ifPresentOrElse(
-                envelope -> logger.info(
-                    "Envelope already processed in system, skipping."
-                        + " ID: {}, filename: {}, container: {}, state: {}",
-                    envelope.id,
-                    envelope.fileName,
-                    envelope.container,
-                    envelope.status.name()
-                ),
+                envelope -> {
+                    if (isStale(envelope)) {
+                        logger.warn(
+                            "Found stale envelope. Resuming its processing. Envelope ID: {}",
```

Comment:
should we also include container and file name?

Reply:
done. (+ some refactoring)